### DISCUSSION TITLE
Take 2 - Couple Ideas For The Hopper

### DISCUSSION THREAD

**Discussion by gitrormeister** [TomoyukiAota/photo-location-map#452] [Ideas]
I thought I posted this earlier...but it's vanished. I've got to ideas for your great little app.
1) Provide the ability to drag and drop images (rather than the whole folder). This would be great for creating a travel timeline of sorts.

and to that end:
2) Add the ability to sort the list of files by Date(time) Taken. Adding a chronology to the images would also be awesome.

Thanks for all the effort on this amazing little tool!

**Comment by TomoyukiAota** [comment]
@gitrormeister
Thanks for the idea.
The feature might not be exactly the same as you described, but some feature to be able to recognize the time can be added as a future enhancement.

**Comment by gitrormeister** [comment]
Thank you for your response. I set the default image handler to "Photos",
from my go-to "FastStone Image Viewer". Neither option opened the files. As
you can see from the short vidcap, I tried clicking both the image and the
icon. It's not critical, just helpful to see smaller details like printed
signs.

Overall, your little app is awesome. I can solve the sort challenge by
using a Bulk Rename Utility, it's just an additional step, but might prove
the best route.

Best,
Ror Martin
